docs/admin/customize.rst

```diff
@@ -43,23 +42,22 @@ The best approach is to create a proper Python package out of your customization
         setup(
             name = "weblate_customization",
             version = "0.0.1",
-            author = "Michal Cihar",
-            author_email = "michal@cihar.com",
+            author = "Your name",
+            author_email = "yourname@example.com",
             description = "Sample Custom check for Weblate.",
-            license = "BSD",
-            keywords = "weblate check example",
+            license = "AGPLv3+",
```

Comment:
```suggestion
            license = "GPLv3+",
```

To be consistent with Weblate. AGPL adds additional requirement for source code download, which Weblate doesn't provide.

Reply:
That doesn't have to be built in. To my knowledge network use differentiates in that it constitutes distribution, which invokes the requirement to share the code. (Per usual means.)

Reply:
The problem to fix here was "BSD" not being a qualified license, as it needs to be stated how many clauses it has.